pxr/usd/pcp/layerStackIdentifier.h

```diff
@@ -77,8 +74,13 @@ class PcpLayerStackIdentifier :
     // Comparison.
     PCP_API
     bool operator==(const This &rhs) const;
```

Comment:
The inline implementations should be on a new line like you've done in all the other changes.

Reply:
Done